boards/st/stm32h573i_dk/stm32h573i_dk.dts

```diff
@@ -307,21 +307,6 @@
 			reg = <0x00000000 DT_SIZE_K(64)>;
```

Comment:
maybe a good idea, to increase the mcuboot partition, now that the other space is no longer used

Reply:
Usually the boot partition is 64K, a next PR could resize but I am not sure this was the purpose here (mainly a bug fix)